packages/webpack/lib/utils/modules.js

```diff
@@ -0,0 +1,12 @@
+const { resolve } = require('path');
+
+module.exports = function getModules() {
+  const NODE_PATH = process.env.NODE_PATH;
+  let modules = ['node_modules'];
```

Comment:
It is very opinionated but I normally prefer a `let` here because i am mutating the array afterwards, I know that it is not technically necessary because the reference remains the same but I just prefer to be explicit.
Anyways, I will follow the `untool` code style :)